### PR TITLE
two quickfixes (check commit msgs)

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -19,10 +19,10 @@ def after_sign_in_path_for(user)
   end
 
   def not_found(msg)
+    binding.pry
```
Comment (gabrielpoca) [inline app/controllers/application_controller.rb]: naps -.-

Comment (naps62) [inline app/controllers/application_controller.rb]: chato -.-

**Comment by azevedo** [comment]
:+1:

**Comment by gabrielpoca** [comment]
:+1:

**Pull request closed without merging**